clusterloader2/testing/density/constraints_100.yaml

```diff
@@ -0,0 +1,21 @@
+heapster:
```

Comment:
fluentd is missing

Reply:
In original density test I found constraints only for "fluentd-elasticsearch" which doesn't appear in clusterloader nor original density test results. I've replaced it with "fluentd-gcp".